nebula_tests/hesai/hesai_ros_decoder_test_main.cpp

```diff
@@ -31,7 +31,8 @@ const nebula::ros::HesaiRosDecoderTestParams TEST_CONFIGS[6] = {
   },
   {
     .sensor_model = "PandarAT128",
-    .return_mode = "LastStrongest",
+    .return_mode = "Dual",
+    .scan_phase = 152,
```

Comment:
We would have to re-record new unit test reference data for AT128 with known `scan_phase` setting to make this a nice number. Because it is unknown to me at the moment, I adjusted it to make the test pass.
Because there is a certain leeway anyway with how to sync to `scan_phase` (see the remarks section of this PR), I don't see a big issue with leaving in this somewhat specific number.